Tools/GNUMake/sites/Make.nrel

```diff
@@ -58,6 +58,14 @@ ifeq ($(USE_MPI),TRUE)
     else ifneq ($(findstring Open MPI, $(shell $(F90) -showme:version 2>&1)),)
       mpif90_link_flags := $(shell $(F90) -showme:link)
       LIBRARIES += $(mpif90_link_flags)
+    else
+      # MPT case (no option available to query link flags)
+      LIBRARIES += -lmpi
+      ifeq ($(USE_CUDA),TRUE)
+        export MPICXX_CXX := nvcc
+        export MPICC_CC   := nvcc
+        export MPIF90_F90 := $(F90)
+      endif
```

Comment:
Maybe we can do this in `Tools/GNUMake/Make.defs` at line 678.  We are already setting `OMPI_CXX`, `MPICH_CXX`, etc.

Reply:
I noticed that, but didn't want to do it if it was too NREL-specific. I can make the change and update the PR.

Reply:
I assume that's Cray's way.  So adding it to `Make.defs` might help others.